web/src/lib/components/validator.tsx

```diff
@@ -29,29 +29,30 @@ export default class Validator extends Component<Props, State> {
     this.loadClip();
   }
 
-  private onVote(vote: boolean) {
-    this.props.api.castVote(this.state.glob, vote).then(() => {
+  private async onVote(vote: boolean) {
+    try {
+      await this.props.api.castVote(this.state.glob, vote);
       this.props.onVote && this.props.onVote(vote);
       this.loadClip();
-    }).catch((err) => {
+    } catch (err) {
       console.error('could not vote on clip from validator', err);
-    });
+    }
   }
 
-  private loadClip() {
+  private async loadClip() {
     this.setState({ loading: true });
-    this.props.api.getRandomClipJson().then((clipJson: ClipJson) => {
-
+    try {
+      const clipJson = await this.props.api.getRandomClipJson();
       this.setState({
         loading: false,
         glob: clipJson.glob,
         sentence: decodeURIComponent(clipJson.text),
         audioSrc: clipJson.sound
       });
-    }, (err) => {
+    } catch (err) {
```

Comment:
you have a space between the `catch` and the opening parenthesis here, while you don't have one on a `catch` further up. please make it consistent :)

Reply:
This reminds me, I really need to add a style linter.

Reply:
This reminds me, I really need to add a style linter.

Reply:
Yes! #444